### PR TITLE
feat(ses): whitelist some recent >= stage 3 additions

### PR DESCRIPTION
**Pull request by erights**

### CODE REVIEW

File: packages/ses/src/whitelist.js [hunk 1]
```diff
@@ -932,14 +935,22 @@ export const whitelist = {
       values: 'boolean',
       // Failed tc39 proposal
       // Seen on FF Nightly 88.0a1
-      at: false,
+      at: 'boolean',
```
Comment (erights) [inline packages/ses/src/whitelist.js:938]: Saw the false, and even though `at` is a withdrawn experiment, I wonder if we should always err on unscopables towards inclusion?

**Comment by mhofman** [comment]
In the latest XS 14.1, I note the following removals:

```
Removing intrinsics.%SharedSymbol%.dispose
Removing intrinsics.%ArrayPrototype%.toReversed
Removing intrinsics.%ArrayPrototype%.toSorted
Removing intrinsics.%ArrayPrototype%.toSpliced
Removing intrinsics.%ArrayPrototype%.with
Removing intrinsics.%ArrayPrototype%.@@unscopables.toReversed
Removing intrinsics.%ArrayPrototype%.@@unscopables.toSorted
Removing intrinsics.%ArrayPrototype%.@@unscopables.toSpliced
Removing intrinsics.%ArrayBufferPrototype%.detached
Removing intrinsics.%TypedArrayPrototype%.toReversed
Removing intrinsics.%TypedArrayPrototype%.toSorted
Removing intrinsics.%TypedArrayPrototype%.with
```

Which seem to all be covered by this PR. 👍

File: packages/ses/src/whitelist.js [hunk 2]
```diff
@@ -932,14 +936,26 @@ export const whitelist = {
...
       // See https://github.com/tc39/proposal-array-find-from-last
       findLast: 'boolean',
       findLastIndex: 'boolean',
+      toReversed: 'booleab',
```
Comment (mhofman) [inline packages/ses/src/whitelist.js:943]:
```suggestion
      toReversed: 'boolean',
```

Comment (erights) [reply packages/ses/src/whitelist.js]: Done

File: packages/ses/src/whitelist.js (hunk 1 above)
Comment (kriskowal) [reply packages/ses/src/whitelist.js:939]: If the property is present on Array.prototype, we certainly want it to be unscopable. But, lockdown will remove it if it is present.

**Review by kriskowal — APPROVED** [review]

File: packages/ses/src/whitelist.js (hunk 1 above)
Comment (erights) [reply packages/ses/src/whitelist.js:939]: I just checked, and we do list `Array.prototype.at` as not to be removed. Given your remarks, it seems that not removing it on unscopables was indeed a correct fix, so Resolving. However, I'm curious: Given

> If the property is present on Array.prototype, we certainly want it to be unscopable.

Shouldn't all methods on Array.prototype already be unscopable in plain JS? Node JS does not even come close:

```
> gopn = Object.getOwnPropertyNames
[Function: getOwnPropertyNames]
> gopn(Array.prototype).length
34
> gopn(Array.prototype[Symbol.unscopables]).length
11
```

Comment (kriskowal) [reply packages/ses/src/whitelist.js:939]: My understanding is that methods of arrays that predates the introduction of unscopables must not be unscopable, and all that followed must be unscopable, such that all legacy code that uses arrays as with block contexts continue to work, both in the case where they intentionally invoke a method of the array and all the cases where a new array method would overshadow a lexical function. 

with (arr) push(10)

function flatMap() {}
with (arr) flatMap()

Comment (erights) [reply packages/ses/src/whitelist.js:939]: Makes sense, thanks!

**Pull request merged**